### PR TITLE
Re-enable  MUSELAB / pxt-wifi-shield

### PR DESCRIPTION
**Pull request by microbit-mark**
Fix has been applied in https://github.com/MUSELAB/pxt-wifi-shield/releases/tag/v1.8.76

### CODE REVIEW

File: targetconfig.json [hunk 1]
```diff
@@ -215,7 +215,7 @@
             "pauldfoster/pxt-microbit-gy521": "dv:mbcodal",
             "pizayanz/pxt-linebeacon": "dv:mbcodal",
             "sparkfun/pxt-gator-environment": "dv:mbcodal",
-            "muselab/pxt-wifi-shield": "dv:mbcodal",
+            "muselab/pxt-wifi-shield": "v1.8.76",
```
Comment (abchatra) [inline targetconfig.json:218]: this need to be "min:v1.8.76" which means minimum version upgrade needs to be done.

**Review by abchatra — CHANGES_REQUESTED** [review]

File: targetconfig.json (hunk 1 above)
Comment (microbit-mark) [reply targetconfig.json]: Ack. Done in https://github.com/microsoft/pxt-microbit/pull/3781/commits/73e5f0ae6f4542ff98e58d9dcecbc91d2fa5cb3d

**Comment by abchatra** [comment]
@microbit-mark this repo has newer tag now. Can you make sure the old tag is ok?

**Comment by microbit-mark** [comment]
I have updated to the latest tagged release

**Review by abchatra — APPROVED** [review]

**Pull request merged**